test/Microsoft.Extensions.SecretManager.Tools.Tests/SecretManagerTests.cs

```diff
@@ -6,37 +6,27 @@
 using System.IO;
 using System.Text;
 using Microsoft.Extensions.Configuration.UserSecrets;
-using Microsoft.Extensions.Configuration.UserSecrets.Tests;
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.SecretManager.Tools;
+using Microsoft.Extensions.Configuration.UserSecrets.Tests;
 using Xunit;
+using Xunit.Abstractions;
 
-namespace Microsoft.Extensions.SecretManager.Tests
+namespace Microsoft.Extensions.SecretManager.Tools.Tests
 {
-    public class SecretManagerTests
+    public class SecretManagerTests : IDisposable
     {
-        [Fact]
-        public void SetSecret_With_ProjectPath_As_CommandLine_Parameter()
-        {
-            SetSecrets(fromCurrentDirectory: false);
-        }
+        private TestLogger logger;
```

Comment:
I don't think this is a good idea - each test should be self-contained. Why do you want to share a logger across multiple tests?